src/openforms/formio/variables.py

```diff
@@ -21,6 +21,8 @@
     "description",
     "html",
     "placeholder",
+    "values",  # types: radio/selectBoxes
+    "data",  # type: select
```

Comment:
```suggestion
    "data",  # component-type: select
```

`#  type:` is a syntax for type annotations [pep 526](https://peps.python.org/pep-0526/) Can you change the one on "groupLabel" too?